core/scss/components/brand-bar/_brand-bar--white.scss

```diff
@@ -0,0 +1,11 @@
+@charset "UTF-8";
+
+#su-brand-bar.su-brand-bar--white {
+  background-color: $color-white;
+
+  .su-brand-bar__logo {
+    color: $color-cardinal-red;
+    -webkit-font-smoothing: auto;
+    -moz-osx-font-smoothing: auto;
```

Comment:
We have a font-smoothing mixin. Would that one work for you? If not, perhaps the mixin can be refactored with variables so that we can pass through whatever value we want. Right now it is:

`@mixin font-smoothing {
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}`

Reply:
I was looking into antialiasing when working on the homesite. The default setting works well on light text over darker background (so for the 3 other variants), but for cardinal red wordmark over white, it kind of needs the opposite setting (which is what I'm adding in this file) for the font weight to look the same in all variants. Otherwise the wordmark looks thinner for this variant. Let me know if there's a better way of handling this.

Reply:
And by "default setting", I meant the font smoothing styles that's currently being bundled into the wordmark logo component which has the same settings as in the font-smoothing mixin.

Reply:
I pushed up a commit with an update to the font-smoothing mixin (I updated the related call in this file, which outputs what we want still). LMK your thoughts.

Reply:
This looks good.